paths/node-runner/README.md

```diff
@@ -42,11 +42,40 @@ If you're determined to learn by doing, we recommend using [Node Pilot](https://
 
 ### Hardware
 
-**Hardware Requirements:** 4 CPU’s (or vCPU’s) | 8 GB RAM | 200GB Disk
+**Hardware Requirements:** 4 CPUs (or vCPUs) | 8 GB RAM | 200GB Disk
+
+Plan hardware for 4 vCPUs per Pocket node, but when creating the node allocate 16. This will allow nodes extra resources to close app large sessions.
+
+Since the Pocket node is a continuously running blockchain, new data is always being stored on it. As 200GB might suffice only for about six months, consider having much more storage than minimally required.  
+
+**NOTE**: You'll also need to run the full nodes of other blockchains, which may have their own hardware requirements that exceed Pocket's. 
+
+The blockchains, i.e., relay chains, you wish to support will determine the vCPU, RAM and disk space you’ll need for the server hosting them. A guide is available [here]: (https://docs.decentralizedauthority.com/hardware-recommendations)
+
+If you plan to run the relay chains on the same machine as your Pocket node, make sure you have enough resources to support them.
+
+TIP: Take the minimum required disk space you calculated for your blockchain node and triple it. This will accommodate potential rapid blockchain growth. As chains and validators tend to get hungrier, e.g., due to unforeseen bugs or natural growth, additional vCPU and RAM may be required later. {: .alert .alert-info}
+
+**OS**: Pocket is officially supported/tested currently on Linux (x86 and aarm) and MacOS. While Pocket nodes are not dependent on the Linux kernel, setting up on the latest Linux LTS release is preferable. 
+
+**SSD** required (get the fastest NVMe possible). Pick a Raid 0 configuration to maximize disk space.
+
+TIP: Chains require constant read/write so this may be your biggest bottleneck. The faster the disk, the better. 
+{: .alert .alert-info}
+
+TIP: Your chains should be as close to your nodes as possible to reduce latency, preferably on the same server. If you’re using multiple servers, have the chains in the same data centre, or at least in the same city.
+{: .alert .alert-info}
+
+When configuring chains - especially on the same server - keep in mind the ports used by each chain and possible conflicts with other chains in your setup. For example, Ethereum clients typically use the same ports as Tendermint-based chains. 
+
+For VPS or Dedicated Servers, you may not have your choice of hardware. So select the closest to the required specifications. You will want as dedicated a server as possible without "noisy neighbours" or shared resources to achieve maximum performance.
+
+**HOST**: Pick a reliable host or data centre. Remember: if they go down - you go down.
```

Comment:
`Remember: if they go down - you go down`. It makes sense, but I'm just not a fan of the wording.

Maybe, "Please note that if validator may get jailed if your host provider is down for a prolonged period of time".